### DISCUSSION TITLE
Load assets from subfolder

### DISCUSSION THREAD

**Discussion by Snoopy137** [AlmasB/FXGL#1263] [Q&A]
Hello, I am trying to load an asset located at /assets/models/DukerZombie/ and it cannot be found, the only way for me to loaded is to move the file to the parent folder, is there a way to solve this?
The code I use is:
Model3D root = getAssetLoader().loadModel3D("/java-duke20.obj");
The folder structure is:

├───main
│   ├───java
│   │   └───com
│   │       └───decssoft
│   │           └───testing3d
│   └───resources
│       └───assets
│           ├───models
│           │   └───DukerZombie
│           └───textures
└───test
    └───java

![imagen](https://user-images.githubusercontent.com/52089562/230149979-2bd6729f-6437-424f-baf9-d84ff8e61d70.png)

**Comment by AlmasB** [comment]
Hi, all 3d models will be searched for under `models`, so `getAssetLoader().loadModel3D("DukerZombie/java-duke20.obj");` should work. If it doesn't please raise an issue with relevant info.

**Comment by ghost** [comment]
It may be my IDE doing this, but I have to add extra lines in "module-info.java" otherwise it says it can't find textures etc.
so you might have to add these two lines:
```
opens assets.models
opens assets.models.DukerZombie
```
To your module-info.java (if you have one!) same goes for Textures and any sub-folders you have in there too.
